### DISCUSSION TITLE
Documentation website down ?

### DISCUSSION THREAD

**Discussion by dfranco** [rectorphp/rector#8397] [General]
Hey, I get a 500 http status code trying to reach https://getrector.com/documentation/

Any issue ?

**Comment by samsonasik** [comment]
Thank you for the report, I am looking at it

**Comment by samsonasik** [comment]
It fixed now, please clear your browser cache and try access again https://getrector.com/documentation/

**Comment by TomasVotruba** [comment]
It's my mistake, I upgraded dependencies without checking the deploy result 

Thank you for fixing @samsonasik

**Reply by dfranco** [reply]
All good now, thanks for the quick fix @samsonasik
